app/controllers/assessments_controller.rb

```diff
@@ -175,6 +176,31 @@ def update
     redirect_to(patient_assessments_url) && return
   end
 
+  def table
+    redirect_to(root_url) if ADMIN_OPTIONS['report_mode']
```

Comment:
Is this really necessary, or just extra?

Reply:
I didn't want anyone to be able to hit this route at all (even if manually entering the url) so it was added out of an abundance of caution. I can remove if we feel it's not necessary at all.

Reply:
Nope, just curious. I think it's fine to leave in!